common/src/main/java/com/microsoft/identity/common/adal/internal/net/HttpWebRequest.java

```diff
@@ -188,21 +188,20 @@ public HttpWebResponse send() throws IOException {
      * @param context Context
      * @throws ClientException throw network exception
      */
-    public static void throwIfNetworkNotAvailable(final Context context) throws ClientException {
+    public static void throwIfNetworkNotAvailable(final Context context, final boolean powerOptCheckEnabled) throws ClientException {
```

Comment:
Added param needs javadoc

Reply:
I might prefer a more descriptive name here. Something along the lines of `skipPowerOptimizationCheck`